### ISSUE TITLE
Pixi: Improve advice given for next gen image formats

### ISSUE THREAD

**Issue by nainar** [ampproject/amp.dev#4699]
Currently we don't list out the resources - that is being tracked in #4666 

However, we should change this advice to be:

Image formats like [AVIF](https://rawcdn.githack.com/AOMediaCodec/av1-avif/67a92add6cd642a8863e386fa4db87954a6735d1/index.html), and WebP often provide better compression than PNG or JPEG, which means faster downloads and less data consumption. [Learn more](https://web.dev/uses-webp-images/?utm_source=lighthouse&utm_medium=unknown).

Consider displaying all your `<amp-img>` components in AVIF and WebP formats while specifying an appropriate fallback (e.g. JPEG 2000, JPEG XR) for other browsers. [Learn more](https://amp.dev/documentation/components/amp-img/#example:-specifying-a-fallback-image).

This means we should no longer depend on PSI for the text. 

@cramforce and @kristoferbaxter since they both have brought this up.

**Comment by cramforce** [comment]
AVIF is too early to recommend. This could change any day now if tooling improves.

The thing with WebP is that the wins over JPEG are often very maginal.

**Comment by kristoferbaxter** [comment]
Perhaps we could still list and display the variants in AVIF format with a deeplink to tooling that supports AVIF format as an example?

This wouldn't necessarily mean validating the document uses AVIF, but including it in the recommendation for when sub-resources are larger than expected using existing rules.

**Comment by patrickkettner** [comment]
@kristoferbaxter once native image support lands, do you imagine recommending &lt;picture&gt; use? It seems like that would allow us to use next gen image formats a lot sooner

**Comment by kristoferbaxter** [comment]
Absolutely, once native image support lands we can recommend using <picture>.

However, this will be quite off since we need to account for all the use-cases of <amp-img>. It's still in the best interest of publishers to support `vary` based on `accept-encoding` and return the correct response type.

**Comment by cramforce** [comment]
In practice using `picture` would make a huge difference. The `vary` stuff just isn't going to happen.
